models/product-type/src/product-type-draft/presets/sample-data-fashion/accessories.ts

```diff
@@ -18,7 +21,7 @@ const accessories = (): TProductTypeDraftBuilder =>
         .inputTip(LocalizedString.presets.empty()['en-US']('Type'))
         .isRequired(true)
         .type(
-          AttributeType.presets.enumType().values([
```

Comment:
Can these attribute type presets be removed?

Reply:
Good call. Much cleaner. See b822e08a81c1dd8d55fbab268185b23a210696d0